app/views/pages/about.html.erb

```diff
@@ -63,7 +63,7 @@
     <div class="about-rows">
       <div class="about-row">
         <div class="about-key">
-          <img src="<%= cloudinary "https://thepracticaldev.s3.amazonaws.com/i/b22fx2whfnyfmm05p856.jpeg" %>" width="250" alt="Ben Halpern" />
+          <%= image_tag "ben-halpern.jpeg", width: 250, alt: "Ben Halpern" %>
```

Comment:
Shouldn't this be a `cl_image_tag` as well?

Reply:
Ignore this because as mentioned there is only a group photo of the founders now.